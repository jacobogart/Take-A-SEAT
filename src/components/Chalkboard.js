import React, { Component } from 'react';
import '../css/Chalkboard.scss';

class Chalkboard extends Component {
  render() {
    return (

    );
  }
}

export default Chalkboard;
