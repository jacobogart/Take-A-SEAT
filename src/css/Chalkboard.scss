.Chalkboard {
  width: 100%;
  height: 100%;
  grid-area: "chalkboard";
}

.phraseContainer {
  font-family: $script-font;
  font-size: 24px;
  color: white;
  text-align: left;
  letter-spacing: 1.5px;
  height: 60vh;
  width: 80%;
  margin: 40px 30px 100px;
  overflow-y: scroll;
  p {
    margin: 10px 0;
  }
}
