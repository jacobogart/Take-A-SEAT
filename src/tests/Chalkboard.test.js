import React from "react";
import Chalkboard from "../components/Chalkboard";
import { shallow } from "enzyme";

let mockCurrentPhrase = "<Chalkboard";
let mockChalkboardPhrases = [
  "let const mockData = {};",
  "let wrapper = shallow("
];

describe('Chalkboard', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Chalkboard
        currentPhrase={mockCurrentPhrase}
        chalkboardPhrases={mockChalkboardPhrases}
      />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
