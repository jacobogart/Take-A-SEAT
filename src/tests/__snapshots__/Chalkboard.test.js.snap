// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chalkboard should match snapshot 1`] = `
<section
  className="Chalkboard"
>
  <div
    className="phraseContainer"
  >
    <p
      key="0"
    >
      let const mockData = {};
    </p>
    <p
      key="1"
    >
      let wrapper = shallow(
    </p>
    <p>
      &lt;Chalkboard
    </p>
  </div>
</section>
`;
